input/fsh/VS_Radiotherapy.fsh

```diff
@@ -198,32 +207,11 @@ Title: "Radiotherapy Treatment Location Qualifier Value Set"
 Description: "Various modifiers that can be applied to body locations where radiotherapy treatments can be directed."
 * ^copyright = "This value set includes content from SNOMED CT, which is copyright © 2002+ International Health Terminology Standards Development Organisation (IHTSDO), and distributed by agreement between IHTSDO and HL7. Implementer use of SNOMED CT is not covered by this agreement"
 * include codes from system SCT where concept is-a #258399006  "Lymph node level value (qualifier value)"  // I, II, III, IV, V, VI, VII and sub-levels
+* exclude SCT#258399006 // the top level abstract code is excluded
```

Comment:
I just wonder if this is the only case where we need to exclude the top-level term in an `is-a` relationship. Is it?